src/Atmos/Model/AtmosModel.jl

```diff
@@ -615,17 +615,18 @@ function. Contributions from subcomponents are then assembled (pointwise).
     aux::Vars,
     t::Real,
 )
+    ρ_pad = SVector(1, 1, 1)
     ρu_pad = SVector(1, 1, 1)
     ts = recover_thermo_state(atmos, state, aux)
     tend = Flux{SecondOrder}()
     args = (atmos, state, aux, t, ts, diffusive, hyperdiffusive)
-    flux.ρ = Σfluxes(eq_tends(Mass(), atmos, tend), args...) .* ρu_pad
+    flux.ρ = Σfluxes(eq_tends(Mass(), atmos, tend), args...) .* ρ_pad
```

Comment:
I am not sure that I follow, but was the change from 'ρu_pad' to 'ρ_pad' accompanied by a change in the definition of 'Σfluxes' ?

Reply:
I just changed this because the previous name was bad. Actually, ideally we shouldn't have this pad at all for `ρ_pad`. Going to see if we can remove it.

Reply:
Going to try just removing this for now.

Reply:
It turns out that we can't remove the padding because, for example, the MMS model doesn't have any second-order fluxes. So we need to ensure that the LHS and RHS ranks match.